gallery/visualtests/nxPolicyViolationIndicator.js

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+const { Target } = require('@applitools/eyes-webdriverio');
+const { simpleTest } = require('./testUtils');
+
+describe('NxPolicyViolationIndicator', function() {
+  beforeEach(async function() {
+    await browser.url('#/pages/NxPolicyViolationIndicator');
+  });
+
+  const policyViolationIndicatorCategoryExampleSelector =
+      '#nx-policy-violation-indicator-category-example .gallery-example-live',
+      policyViolationIndicatorNumberExampleSelector =
+        '#nx-policy-violation-indicator-number-example .gallery-example-live';
+
+  it('looks right', simpleTest(policyViolationIndicatorCategoryExampleSelector));
+  it('looks right', simpleTest(policyViolationIndicatorNumberExampleSelector));
```

Comment:
I don't think this second one is necessary.  Keep in mind visual tests are just about different visual variations.  Different data or usage variations would be covered by unit tests.

Reply:
> I don't think this second one is necessary. Keep in mind visual tests are just about different visual variations. Different data or usage variations would be covered by unit tests.

Ah, removed: https://github.com/sonatype/sonatype-react-shared-components/pull/229/commits/f2ced0972a54e081b9882d395dffbd79bfa59f0c

Reply:
> Ah, removed: f2ced09

I believe you've got an unused variable now.

Reply:
> > Ah, removed: [f2ced09](https://github.com/sonatype/sonatype-react-shared-components/commit/f2ced0972a54e081b9882d395dffbd79bfa59f0c)
> 
> I believe you've got an unused variable now.

https://github.com/sonatype/sonatype-react-shared-components/pull/229/commits/4417f2cb8012dcea9fea8fdbc3f37675b97209dd